tern/analyze/docker/analyze.py

```diff
@@ -106,7 +106,12 @@ def analyze_subsequent_layers(image_obj, shell, master_list, redo, dfobj=None,
                               dfile_lock=False):
     # get packages for subsequent layers
     curr_layer = 1
+    #set current work_dir to none
+    work_dir = None
     while curr_layer < len(image_obj.layers):  # pylint:disable=too-many-nested-blocks
+        #update workdir
+        if image_obj.layers[curr_layer].get_layer_workdir():
+            work_dir = image_obj.layers[curr_layer].get_layer_workdir()
```

Comment:
Instead of making the same function call twice in a row -- what about just saying `work_dir = image_obj.layers[curr_layer].get_layer_workdir()` ?